src/features/Search/constants/search-types.js

```diff
@@ -0,0 +1,3 @@
+export const STOCKS = 'STOCKS';
+export const CURRENCIES = 'CURRENCIES';
+export const CRYPTOCURRENCIES = 'CRYPTOCURRENCIES';
```

Comment:
Here we define constants for our identifiers, declaring them in their own file, make this a source of truth for the code.
If we change something here it will take place in all other components which use these variables, it makes the code easier to maintain.
If (more like when) you start working with TypeScript you can than also define these variables as a type and validate it when passing props